packages/ui/src/composites/wui-icon-box/index.ts

```diff
@@ -31,18 +32,21 @@ export class WuiIconBox extends LitElement {
         iconSize = 'xxs'
     }
 
-    const classes = {
-      [`wui-size-${this.size}`]: true,
-      [`wui-overlay-${this.size === 'lg' ? 'sm' : 'md'}`]: true
-    }
-
-    const bgColorStyle = `--wui-bg-value: var(--wui-color-${this.backgroundColor});`
-
-    return html`
-      <div class="wui-overlay-default ${classMap(classes)}" style=${bgColorStyle}>
-        <wui-icon color=${this.iconColor} size=${iconSize}><slot></slot></wui-icon>
-      </div>
-    `
+    this.style.cssText = `
```

Comment:
This looks a bit odd / inconsistent with everything else we have going on, plus I think we lost color-mix fallback of `--wui-overlay-020` that used to be here before. Previous approach with styles being in styles.ts looked more readable / cleaner.

What if instead of doing full style here, you just set css variables and still use styles in their own files? i.e. set up all variable options as local css variables (let's reserve --wui prefix for global) and then use them inside styles.ts

```ts
   const isLg = this.size === lg
   const bgMix = isLg ? '12%' : '16%'
   const borderRadius =  isLg ? '3xl' : 'xxs'
   
    this.style.cssText = `
      --local-bg-value: var(--wui-color-${this.backgroundColor});
      --local-bg-mix: ${bgMix};
      --local-border-radius: var(--wui-border-radius-${borderRadius});
      --local-size: var(--wui-icon-box-size-${this.size});
  `
```